src/index.html

```diff
@@ -7,6 +7,36 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day start-day_sun"></div>
```

Comment:
`start-day_sun` should be a modifier.

More about naming you could watch  [there](https://mate.academy/learn/css-advanced/bem#/video/920)
timing 0:30

```suggestion
    <div class="calendar">
      <div class="calendar__day calendar__day--start-day_sun"></div>
```